.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - "*"
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  ci:
+    name: ${{ matrix.task.name}} - ${{ matrix.python.name }}
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - name: CPython 2.7
+            tox: py27
```

Comment:
do we need tox ?

maybe just 
```
python_version: [2.7, 3.9, pypy3]
```

Reply:
There was no plan of using tox, so yes, that should go away.  Thanks.  You've probably noticed that I have a habit of fairly verbose and explicit matrixing.  I'm used to working on things like pytest-twisted and pyqt5-tools etc that have significant matrixing and where I've bit myself a few times by not just giving a unique value for each use.  So, I'll probably keep the explicit name here.